src/main/java/uk/gov/hmcts/reform/juddata/camel/task/LeafRouteTask.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.juddata.camel.task;
+
+import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.scope.context.ChunkContext;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.batch.repeat.RepeatStatus;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LeafRouteTask implements Tasklet {
+
```

Comment:
we can also add implements StepExecutionListener to  process any step before or after events
This will also give flexibility if we need to extend before and after events in future